app/views/payments/create.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Payments#create</h1>
+<p>Find me in app/views/payments/create.html.erb</p>
```

Comment:
**OPTIONAL** - Please remove this view file since you have not made use of it in your application, it is important to keep your code clean by leaving only needed files in your project. Please fix this 😄